kahuna/public/js/leases/leases.js

```diff
@@ -183,9 +183,8 @@ leases.controller('LeasesCtrl', [
         ctrl.leaseStatus = (lease) => {
             const active = lease.active ? 'active ' : ' ';
 
-            const current = ctrl.leases.current
-                .filter((lease) => lease !== null)
-                .find(l => l.id == lease.id) ? 'current ' : '';
+            // Current only makes sense for use leases
+            const current = (lease.active && lease.access.match(/-use/i)) ? 'current' : '';
```

Comment:
Maybe add a field to the lease to indicate this rather than regex matching?

Reply:
Do you mean, have a `current` boolean field for the lease?

Reply:
Yeah something like that

Reply:
A flag meaning `isUse`